components/organisms/Header.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from "prop-types";
+import Image from "next/image";
+import { SearchBar } from "../atoms/SearchBar";
+import { useState } from "react";
+
+export function Header(props) {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
+
+  return (
+    <div className="w-full  mt-6 ">
+      <div className="h-auto flex  md:flex-row justify-center  mt-4">
+        <div className="flex  flex-col  lg:flex-col  xl:flex-row container mx-auto justify-between relative">
+          <div className="md:h-9 h-6 mb-4">
+            <Image
+              className="h-full  bottom-0 left-0"
+              width={300}
+              height={80}
+              src={props.headerLogoImage}
+              alt={props.headerLogoAltText}
+            />
+          </div>
+
+          <div>
+            <SearchBar
+              placeholder={"Search Canada.ca"}
+              onChange={handleInputChange}
+              dataCy={"search-bar"}
+            />
+          </div>
```

Comment:
On Canada.ca, the search bar is full-width on mobile.

<img width="599" alt="Screen Shot 2021-04-07 at 14 34 39" src="https://user-images.githubusercontent.com/2454380/113916845-7b14da80-97ae-11eb-8ac3-0434a701c4fa.png">